example_test.go

```diff
@@ -15,6 +15,11 @@ func ExampleNewFinder() {
 		Extensions: []string{".ttf"},
 	})
 
+	// Create a new finder only searches the current directory for fonts.
```

Comment:
Please rephrase the comment a bit. Something like:
`Create a new finder which searches for fonts only in the current directory.`